logstash/pipeline/logstash.conf

```diff
@@ -1,7 +1,8 @@
 input {
-	tcp {
-		port => 5000
-	}
+    tcp {
+        port  => 5000
+        codec => json
```

Comment:
Well, my comment about Filebeat was actually not accurate because `beats` requires a special input on the Logstash side. However instead of altering the default tcp input I'll mention the requirement inside a README file (eg. inside extras/logspout/README.md). How does it sound?